operator/src/main/java/oracle/kubernetes/operator/helpers/JobHelper.java

```diff
@@ -92,18 +91,20 @@ Domain getDomain() {
     }
 
     @Override
-    List<V1EnvVar> getEnvironmentVariables(TuningParameters tuningParameters) {
-      List<V1EnvVar> envVarList = new ArrayList<V1EnvVar>();
-      addEnvVar(envVarList, "NAMESPACE", getNamespace());
-      addEnvVar(envVarList, "DOMAIN_UID", getDomainUID());
-      addEnvVar(envVarList, "DOMAIN_HOME", getDomainHome());
-      addEnvVar(envVarList, "NODEMGR_HOME", getNodeManagerHome());
-      addEnvVar(envVarList, "LOG_HOME", getEffectiveLogHome());
-      addEnvVar(envVarList, "INTROSPECT_HOME", getIntrospectHome());
-      addEnvVar(envVarList, "SERVER_OUT_IN_POD_LOG", getIncludeServerOutInPodLog());
-      addEnvVar(envVarList, "CREDENTIALS_SECRET_NAME", getWebLogicCredentialsSecretName());
-
-      return envVarList;
+    List<V1EnvVar> getConfiguredEnvVars(TuningParameters tuningParameters) {
+      // Pod for introspector job would use same environment variables as for admin server
+      List<V1EnvVar> vars = getDomain().getAdminServerSpec().getEnvironmentVariables();
```

Comment:
Don't you need to make a copy of the AdminServerSpec environment variable list before adding more vars?